linera-base/src/data_types.rs

```diff
@@ -718,34 +718,34 @@ pub struct ApplicationPermissions {
     /// If it is `Some`, only operations from the specified applications are allowed, and
     /// no system operations.
     #[debug(skip_if = Option::is_none)]
-    pub execute_operations: Option<Vec<UserApplicationId>>,
+    pub execute_operations: Option<Vec<MultiAddress>>,
     /// At least one operation or incoming message from each of these applications must occur in
     /// every block.
     #[graphql(default)]
     #[debug(skip_if = Vec::is_empty)]
-    pub mandatory_applications: Vec<UserApplicationId>,
+    pub mandatory_applications: Vec<MultiAddress>,
```

Comment:
ok so afaict this one makes no sense for user addresses

Reply:
There's no such thing as "user addresses" anymore. It's just address (both for accounts and applications - system and user).